homeassistant/components/amberelectric/translations/en.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Wait this is weird. Where is `strings.json` ? This file is automatically generated based on `strings.json`.